### PR TITLE
Update shufflenetv2.py

### PR DESCRIPTION
**Pull request by 486755496**
"from .utils import load_state_dict_from_url": The "." should be removed.

### CODE REVIEW

**Comment by NicolasHug** [comment]
Thanks for the PR @486755496 but I think the current use of `.` (relative import) is correct. The proposed changed is causing multiple CI failures,
